controller/array_action/array_mediator_ds8k.py

```diff
@@ -266,7 +266,7 @@ def _create_api_volume(self, name, size_in_bytes, array_space_efficiency, pool_i
             )
             raise array_errors.VolumeCreationError(name)
 
-    def create_volume(self, name, size_in_bytes, space_efficiency, pool, io_group):
+    def create_volume(self, name, size_in_bytes, space_efficiency, pool, io_group, volume_group):
```

Comment:
don't it better to add a default to `volume_group` parameter in order to avoid `None` passes like [here](https://github.com/IBM/ibm-block-csi-driver/blob/7610c0643d2d44d7f3dfdeeaf04b7ba913929639/controller/tests/array_action/ds8k/test_array_mediator_ds8k.py#L198)?

Reply:
never mind